### PR TITLE
utils/utils.py: Use bot configuration

### PR DESCRIPTION
**Pull request by LifetimeAlone**
utils/utils.py: Use bot configuration

Use COBOT_ROOT variable set in config.py
instead of accessing enviroment.

Fixes https://github.com/coala/corobo/issues/384

# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: utils/utils.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 from errbot import BotPlugin, botcmd
 
+import config
```
Comment (jayvdb) [inline utils/utils.py]: @meetmangukiya , is this OK.  I vaguely recall you had concerns about `config` being imported.

Comment (meetmangukiya) [reply utils/utils.py]: Yes, please remove this import statement, and use `self.bot_config.COBOT_ROOT` instead

**Comment by jayvdb** [comment]
unack https://github.com/coala/corobo/pull/414/commits/72e363cfb8989d43283ad84a2e3b63b36e6e5180

**Comment by Vamshi99** [comment]
ack e678776

**Comment by meetmangukiya** [comment]
@rultor merge

**Pull request merged**